modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -11,19 +11,74 @@ image::scatter-gather-concept-56ccc.png[]
 
 WARNING: A Scatter-Gather component cannot be configured with only one route. A Mule application will throw an exception and not start if it contains a flow in which a Scatter-Gather component has not been configured with at least two routes.
 
-## How Errors Generated Inside Scatter-Gather Routes are Handled
+== How Errors Generated Inside Scatter-Gather Routers are Handled
```

Comment:
This should be "routes" as is, since it is not trying to explain for the entire router, but for each route (since some could raise an error, and others couldn't)

Reply:
The same error was in v4.2, I brought this change from there. I'll fix in both versions.

Reply:
👍